### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by maecapozzi**
Continued attempt at updating CONTRIBUTING.md in the hopes that it will clarify how to set up the dev environment for new contributors.

### CODE REVIEW

**Comment by maecapozzi** [comment]
I'd love some feedback on why my build would be failing while making documentation changes!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,12 +10,17 @@ https://www.vagrantup.com/downloads.html
...
   * If changes have been made since running vagrant up: <code>vagrant provision</code>
-  * To login to the machine: <code>vagrant ssh</code>
-  * `cd  /vagrant/` to navigate to the refuge repo.
-  * There is a rake wrapper to execute commands in refuge repo on the vagrant machine
-  without using ssh. Run `rake vagrant:shell[command]`
-  * To start the rails server bind it to the vagrant machine's IP address after running `vagrant ssh`, use: `rails s -b 0.0.0.0`. Otherwise, run: `rake vagrant:shell['rails s -b 0.0.0.0']`
-  * The website is available at `localhost:3000`
+  
+  Please note that there are two ways to run commands in vagrant:
+  1. A local `rake` wrapper that allows direct execution on the machine.
```
Comment (mi-wood) [inline CONTRIBUTING.md]: For some reason this isn't rendering as a numbered list. It may have to be indented to recognize a change in list type

**Comment by mi-wood** [comment]
Thanks @maecapozzi! Just one minor comment. Looks like the build passed

**Comment by tkwidmer** [comment]
This looks good to me Thanks @maecapozzi

**Pull request merged**